docs/devel/storage.md

```diff
@@ -76,7 +76,7 @@ The CDI controller will see the annotation and spawn an 'importer' pod that does
 
 Now that we have a populated PVC, we can create a VM using the disk image. For instance using this yaml:
 ```YAML
-apiVersion: kubevirt.io/v1alpha1
+apiVersion: kubevirt.io/v1alpha2
 kind: VirtualMachine
```

Comment:
IIUC, VirtualMachine needs to be changed to VirtualMachineInstance

Reply:
No, VMs are what used to be OfflineVMs, and VM instances are what used to be VMs. Were are defining the specification of the VM here, so the kind should be Virtual Machine, when we start this VMs, we in essence create a VM instance.

Reply:
I tried running the example just a minute ago and got the following error:
```
[thiago@localhost kubevirt]$ ./cluster/kubectl.sh create -f /tmp/vm.yaml 
error: unable to recognize "/tmp/vm.yaml": no matches for kind "VirtualMachine" in version "kubevirt.io/v1alpha1"
```
Then while checking the kubevirt examples[1] I notcied that it had changed to VirtualMachineInstance. Making this change fixed it for me.

[1] - https://github.com/kubevirt/kubevirt/blob/master/cluster/examples/vmi-ephemeral.yaml

Reply:
Right, that is why this PR is changing it to v1alpha2. Again VirtualMachine is the definition, and VirtualMachineInstance is the running instance of a VirtualMachine. The example you are looking at is creating a running VM, this example is like [1]. Creating the definition and then you can launch it at a later time.

[1] https://github.com/kubevirt/kubevirt/blob/master/cluster/examples/vm-cirros.yaml

Reply:
No you are right, I was confused, it should be a VirtualMachineInstance, I have updated the PR to match.